jobserver/models.py

```diff
@@ -349,30 +350,67 @@ def __str__(self):
 
 class Org(models.Model):
     name = models.TextField(unique=True)
+    slug = models.SlugField(max_length=255, unique=True)
 
     class Meta:
         verbose_name = "Organisation"
 
     def __str__(self):
         return self.name
 
+    def get_absolute_url(self):
+        return reverse("org-detail", kwargs={"org_slug": self.slug})
+
+    def save(self, *args, **kwargs):
+        if not self.slug:
+            self.slug = slugify(self.name)
+
+        return super().save(*args, **kwargs)
+
 
 class Project(models.Model):
     org = models.ForeignKey(
         "Org",
         on_delete=models.CASCADE,
         related_name="projects",
     )
+    researcher_registrations = models.ManyToManyField(
+        "ResearcherRegistration",
+        related_name="projects",
+    )
 
-    name = models.TextField(blank=True)
-    display_name = models.TextField(blank=True)
-    email = models.TextField(blank=True)
-    project_lead = models.TextField(blank=True)
+    name = models.TextField(unique=True)
+    slug = models.SlugField(max_length=255, unique=True)
     proposed_start_date = models.DateTimeField(null=True, blank=True)
     proposed_duration = models.TextField(blank=True)
 
-    # store a link to our modified FormA
-    form_url = models.TextField(blank=True)
+    # Section 1
```

Comment:
Worth explaining what "section 1" refers to (i.e. NHSE Form B?)

Reply:
I've shifted the registration fields to the bottom of the model fields and explained what they all, linking to our Form A template, that should given enough context I think? (hope?)